docs/transformers/boolean-converter.md

```diff
@@ -0,0 +1,30 @@
+<span style="float:right;"><a href="https://github.com/RubixML/ML/blob/master/src/Transformers/BooleanConverter.php">[source]</a></span>
+
+# Boolean Converter
+This transformer is used to convert boolean values to a compatible continuous or categorical datatype. Strings should be
+used when the boolean should be treated as a categorical value. Ints or floats when the boolean should be treated as a
+continuous value.
+
+**Interfaces:** [Transformer](api.md#transformer)
+
+**Data Type Compatibility:** Categorical, Continuous
+
+## Parameters
+| # | Name | Default | Type | Description |
+|---|---|---|---|---|
+| 1 | trueValue | 'true' | string, int, float | The value to convert `true` to. |
+| 2 | falseValue | 'false' | string, int, float | The value to convert `false` to. |
+
+## Example
+```php
+use Rubix\ML\Transformers\BooleanConverter;
+
+$transformer = new BooleanConverter($trueValue = 'true', $falseValue = 'false);
```

Comment:
Serious question, is this correct syntax for PHP 8 named arguments?

Even if it is, maybe use the old school method of injecting constructor parameters for now until PHP really catches on

Reply:
From: https://stitcher.io/blog/php-8-named-arguments

Example of named arguments in PHP 8. 
```
setcookie(
    name: 'test',
    expires: time() + 60 * 60 * 2,
);
```